requirements/tests.txt

```diff
@@ -1,2 +1,7 @@
 -r base.txt
-nose
\ No newline at end of file
+nose
+codecov
+coverage
+requests_mock
+nose_parameterized
+janis-pipelines.runner>=v0.11.0
```

Comment:
We might need a different "unittests" and "testing" requirements files.
This one is being used for github actions is failing before it runs, but is it really required?